src/pages/partners/affiliate-ib/_deriv-ib-programme.tsx

```diff
@@ -182,16 +171,7 @@ const DerivIBProgramme = () => {
                         >
                             {localize('Can’t decide which programme or commission plan suits you?')}
                         </StyledHeader>
-                        <LinkButtonContactUsWrapper>
-                            <LinkButtonContactUs
-                                external
-                                secondary
-                                to="mailto:partners@deriv.com"
-                                is_mail_link
-                            >
-                                {localize('Contact us')}
-                            </LinkButtonContactUs>
-                        </LinkButtonContactUsWrapper>
+                        <LiveChatLinkText text="Contact us via live chat" weight="bold" />
```

Comment:
```suggestion
    <LiveChatLinkText text="_t_Contact us via live chat_t_" weight="bold" />
```
since you used `text` directly in `LiveChatLinkText`, you have to wrap the text with `_t_`.